app/src/main/java/com/censocustody/android/common/util/BioCryptoUtil.kt

```
@@ -29,6 +43,7 @@ object BioCryptoUtil {
         BiometricPrompt(fragmentActivity, object : BiometricPrompt.AuthenticationCallback() {
             override fun onAuthenticationError(errorCode: Int, errString: CharSequence) {
                 super.onAuthenticationError(errorCode, errString)
```

When authentication error occurs during biometry prompt, we want to report the error